usgs-details.html

```diff
@@ -239,14 +292,21 @@
             xhr.send();
         }
 
+
+
         // Get metric from Query Parameter and fetch and populate values
         window.onload = function() {
             var metric = METRICS[getQueryParameterByName('metric')];
             if (!metric) {
                 console.warn('Could not find metric for ' + metric);
+                
+                // Sensor lacks metrics, so giving its div.container a name of "sensor"
+                // in order to apply some specific styles.
+                var container = document.getElementsByClassName('container');
+                container[0].setAttribute('name', 'sensor');
```

Comment:
That makes sense! I'm not super sure how to go about doing that (my javascript skills are that I can _mostly_ read it, haha) so the revisions to the commit I put up won't include that.